packages/hs-react-ui/src/components/TextInput/TextInput.tsx

```diff
@@ -176,24 +154,15 @@ const TextInput = ({
       <InputComponent
         cols={cols}
         rows={rows}
-        aria-label={ariaLabel}
-        placeholder={placeholder}
         onChange={(e: SyntheticEvent) => {
           e.persist();
           onChange(e);
```

Comment:
looks like onChange is removed above, this should pull from the rest object
```suggestion
          nativeInputHTMLAttributes.onChange(e);
```

Reply:
I ended up adding the onChange back to the api, as it wasn't being recognized.